rasa/core/actions/action.py

```diff
@@ -178,10 +215,13 @@ class ActionBack(Action):
     def name(self) -> Text:
         return ACTION_BACK_NAME
 
-    def run(self, dispatcher, tracker, domain):
+    async def run(self, output_channel, nlg, tracker, domain):
```

Comment:
wasn't sure if this async should be added -- commenting to make sure it doesn't get lost in review.